src/main/jte/bookings/single.jte

```diff
@@ -66,14 +75,14 @@
     <div class="card-actions justify-end">
         @if (allowEditing)
             @template.util.form(csrf = csrf, action = "/" + booking.getRoom().getId() + "/bookings/" + booking.getId(), method = DELETE, content = @`
-                <button type="submit" class="btn btn-ghost">
+                <button type="submit" class="btn btn-ghost" aria-label="${context.localize("bookings.tooltip.delete")}" title="${context.localize("bookings.tooltip.delete")}">
                     <i class="fas fa-trash-alt"></i>
                     ${context.localize("bookings.delete")}
                 </button>
             `)
         @endif
         @if (showRequestButton)
-            <a class="btn btn-primary" href="/${booking.getRoom().getId()}/bookings/new?start=${booking.getStartDate().format(DateTimeFormatter.ISO_DATE_TIME)}&end=${booking.getEndDate().toLocalTime().format(DateTimeFormatter.ISO_LOCAL_TIME)}&cooperative=true">${context.localize("bookings.request")}</a>
+            <a class="btn btn-primary" href="/${booking.getRoom().getId()}/bookings/new?start=${booking.getStartDate().format(DateTimeFormatter.ISO_DATE_TIME)}&end=${booking.getEndDate().toLocalTime().format(DateTimeFormatter.ISO_LOCAL_TIME)}&cooperative=true" aria-label="${context.localize("bookings.tooltip.request")}" title="${context.localize("bookings.tooltip.request")}">${context.localize("bookings.request")}</a>
```

Comment:
`bookings.request.tooltip` or similar would be nicer.
Also, don't repeat it for both `aria-label` and `title`